courses/free-courses-ar.md

```diff
@@ -22,7 +25,10 @@
   * [PugJs](#pugjs)
   * [React.js](#reactjs)
   * [Vue.js](#vuejs)
+* [Logic Programming - Prolog](#logic-programming---prolog)
```

Comment:
In english list by lang each one has it own category: `Prolog`, `Haskell`, `Lisp`...